tasks/config/licenses.js

```diff
@@ -46,6 +46,7 @@ module.exports = function () {
         'css-stringify@1.0.5': ['MIT'],
         'css@1.0.8': ['MIT'],
         'delegate@3.0.1': ['MIT'],
+        'elasticsearch-browser@13.0.1': ['Apache-2.0'],
```

Comment:
The license checker is looking for `Apache License` (see https://github.com/davglass/license-checker/blob/de9710b0ed7a17265e352a23e66bcc96eba956c7/lib/license.js#L13), but we only specify `Apache 2.0` in the package.json (see https://github.com/elastic/bower-elasticsearch-js/blob/master/package.json). I just want to get this in, so imo it's something we can fix later.

Reply:
I'm not following, we specify "Apache-2.0" above as a valid license, and this seems to pass locally without this modification.

Reply:
This is overriding what we receive from the license checker dependency (which wasn't able to find a valid license for the `elasticsearch-browser` dep).

So we're first running it through the `license-checker` dep, which basically just tries to find the license for every dep, then we're running it through some of our code:  https://github.com/elastic/kibana/blob/master/tasks/lib/licenses/valid.js#L33, which is checking against the licenses specified in this array: https://github.com/elastic/kibana/blob/master/tasks/config/licenses.js#L4.

Reply:
It's failing now, I'm not sure what I did differently earlier. 

The current behavior of parsing the README does make me pause. Are we confident that upgrading this library is what we want if it's regressed and can't determine the license of the elasticsearch-browser package which appears to be pretty normal. Any idea why it doesn't like this one in particular?

Reply:
Good point. I'll close this and the other PR for now, and we can bring it back later. I don't have time to explore the internals of the dep right now, so I'm not sure why it regressed.